app/views/categories/index.html.erb

```diff
@@ -13,42 +13,32 @@
   </div>
 </div>
 </div>
-    
-
   <div class="row">
     <% @categories.each do |category| %>
       <div class="col-lg-4 col-md-6 col-sm-12 mb-4">
-        <div class="car">
-          <div class="total-transactions">
-            <h2>Total Amount: $<%= category.total_amount %></h2>
-          </div>
-          <div class="card-header">
-            <b><%= link_to category.name, category_transactions_path(category) %></b>
-          </div>
-          <div class="card-body">
-            <table class="table">
-              <thead>
-                <tr>
-                  <th>Icon</th>
-                  <th>Name</th>
-                  <th>Amount</th>
-                </tr>
-              </thead>
-              <tbody>
-                <% category.transactions.each do |transaction| %>
-                  <tr>
-                    <td><img src="<%= category.icon %>" alt="icon"></td>
-                    <td><%= transaction.name %></td>
-                    <td>$<%= transaction.amount %></td>
-                  </tr>
-                <% end %>
-              </tbody>
-            </table>
+          <%# <div class="table"> %>
+           <%# <table class="table"> %>
+  <table class="table">
+  <tbody>
+    <tr>
+      <td class="square-cell">
+        <img src="<%= category.icon %>" alt="icon">
+      </td>
+      <td class="l">
+        <%= link_to category.name, category_transactions_path(category), class: "blue-link" %>
+      </td>
+      <td class="square-cell align-c">
+        <b>$<%= category.total_amount %></b>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
           </div>
         </div>
       </div>
     <% end %>
   </div>
 <div class="fixed-bottom">
-  <%= link_to 'New category', new_category_path, class: 'btn btn-primary btn-lg bg-success d-block w-100' %>
+ <%= link_to 'New category', new_category_path, class: 'btn btn-primary btn-lg d-block w-100', style: 'background-color: #5fb523; padding: 20px; font-size: 24px;' %>
```

Comment:
[OPTIONAL] Its a good practice to avoid inline styling. This makes updates to your code easier